product_meat_unece/models/unece_code_list.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class UneceCodeList(models.Model):
+    _inherit = "unece.code.list"
+
+    type = fields.Selection(
```

Comment:
IMO a field named `type` is not correct because `type` is a python's reserved word.

Reply:
I agree, but that's defined in `base_uence`, which is merged in `13.0` already. That can be refactored in a separated PR.